examples/3d/3d_shapes.rs

```diff
@@ -17,10 +18,18 @@ fn main() {
     App::new()
         .add_plugins((
             DefaultPlugins.set(ImagePlugin::default_nearest()),
+            #[cfg(any(not(target_arch = "wasm32"), feature = "webgpu"))]
             WireframePlugin,
         ))
         .add_systems(Startup, setup)
-        .add_systems(Update, (rotate, toggle_wireframe))
+        .add_systems(
+            Update,
+            (
+                rotate,
+                #[cfg(any(not(target_arch = "wasm32"), feature = "webgpu"))]
```

Comment:
why not just exclude wasm32?

Reply:
I thought that `POLYGON_MODE_LINE` was supported in WebGpu but its not. Will remove this.